net-im/spectrum2/spectrum2-2.0.10.1-r2.ebuild

```diff
@@ -49,11 +49,11 @@ RDEPEND="
 	whatsapp? ( net-im/transwhat )"
 
 DEPEND="
-	${PYTHON_DEPS}
 	${RDEPEND}
 	doc? ( app-doc/doxygen )
 	test? (
-		dev-python/sleekxmpp[${PYTHON_USEDEP}]
+		${PYTHON_DEPS}
+		dev-python/sleekxmpp
```

Comment:
why did you drop `${PYTHON_USEDEP}` here?

Reply:
Because that was the cause of the repoman failure?

Reply:
devmanual's got you covered: https://wiki.gentoo.org/wiki/Project:Python/python-any-r1#Package_with_additional_dependencies

Reply:
Ok, please have a look, if this is now correct.